pkg/controller/render/render_controller.go

```diff
@@ -430,18 +430,27 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		return err
 	}
 	if len(mcs) == 0 {
-		return fmt.Errorf("no MachineConfigs found matching selector %v", selector)
+		return ctrl.syncFailingStatus(pool, fmt.Errorf("no MachineConfigs found matching selector %v", selector))
 	}
 
 	if err := ctrl.syncGeneratedMachineConfig(pool, mcs); err != nil {
 		return ctrl.syncFailingStatus(pool, err)
 	}
 
+	return ctrl.syncAvailableStatus(pool)
+}
+
+func (ctrl *Controller) syncAvailableStatus(pool *mcfgv1.MachineConfigPool) error {
+	sdegraded := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolRenderDegraded, v1.ConditionFalse, "", "")
+	mcfgv1.SetMachineConfigPoolCondition(&pool.Status, *sdegraded)
+	if _, err := ctrl.client.MachineconfigurationV1().MachineConfigPools().UpdateStatus(pool); err != nil {
+		return err
+	}
 	return nil
 }
 
 func (ctrl *Controller) syncFailingStatus(pool *mcfgv1.MachineConfigPool, err error) error {
-	sdegraded := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolDegraded, v1.ConditionTrue, fmt.Sprintf("Failed to render configuration for pool %s: %v", pool.Name, err), "")
+	sdegraded := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolRenderDegraded, v1.ConditionTrue, fmt.Sprintf("Failed to render configuration for pool %s: %v", pool.Name, err), "")
 	mcfgv1.SetMachineConfigPoolCondition(&pool.Status, *sdegraded)
 	if _, updateErr := ctrl.client.MachineconfigurationV1().MachineConfigPools().UpdateStatus(pool); updateErr != nil {
 		glog.Errorf("Error updating MachineConfigPool %s: %v", pool.Name, updateErr)
```

Comment:
lower?

Reply:
oh it's a log so we want logs to start uppercase (when it's `fmt.Errorf` or `errors.New` we want them lowercase cause we might end up bubbling up and concatenating)

Reply:
ah then transfer this comment over to all of the utilruntime ones like:
`utilruntime.HandleError(fmt.Errorf("Couldn't get object from tombstone %#v", obj))`

Reply:
will address this later :smile: